src/gdb/gdb.c

```diff
@@ -144,32 +145,74 @@ void gdb_handle_exception(void)
 void parse_request(void)
 {
 	unsigned char *request;
+	unsigned int i;
 	int addr;
-
-	while (1) {
-		request = get_packet();
-		/* Log any exception caused by debug exception */
-		gdb_debug_info(request);
-
-		/* Pick incoming request handler */
-		unsigned char command = *request++;
-
-		switch (command) {
-		/* Continue normal program execution and leave debug handler */
-		case 'c':
-			if (hex_to_int(&request, &addr))
-				sregs[DEBUG_PC] = addr;
-
-			/* return from exception */
-			return;
+	int length;
+
+while (1) {
```

Comment:
indentation looks wrong.

Reply:
@lgirdwood @mrajwa 
Move loop body to another function (may be static inline if you care about call performance penalty), then you'll keep indent of that part

Reply:
Imho still unresolved. Please change it.